packages/entities/entities-plugins/src/components/free-form/Datakit/flow-editor/store/store.ts

```diff
@@ -49,12 +49,33 @@ const [provideEditorStore, useOptionalEditorStore] = createInjectionState(
           state.value.nodes.map((node) => [node.name, node]),
         ),
     )
-    const edgeMapById = computed(
-      () =>
-        new Map<EdgeId, EdgeInstance>(
-          state.value.edges.map((edge) => [edge.id, edge]),
-        ),
+    const edgeMaps = computed(
+      () => {
+        const edgeMapById = new Map<EdgeId, EdgeInstance>()
+        const edgeIdMapByNodeId = new Map<NodeId, Set<EdgeId>>()
+
+        state.value.edges.map((edge) => {
```

Comment:
nit: we can use `for ... of` instead, since we don't really need the result of `map`